CommandLine/emake/Makefile

```diff
@@ -17,7 +17,7 @@ LDFLAGS   += $(OS_LIBS) -L../../ -lcompileEGMf -lEGM -lProtocols -lENIGMAShared
 
 ifeq ($(TESTS), TRUE)
 	TARGET=../../emake-tests
-	SOURCES := $(call rwildcard, ../emake-tests,*.cpp) $(call rwildcard, ../emake-tests/Extensions/Json,*.cpp)
+	SOURCES := $(call rwildcard, ../emake-tests,*.cpp) $(call rwildcard, ../emake-tests/Extensions/Json,*.cpp) $(call rwildcard, ../../ENIGMAsystem/SHELL/Universal_System,visual_shader.cpp) $(call rwildcard, ../../ENIGMAsystem/SHELL/Universal_System,visual_shader_nodes.cpp)
```

Comment:
@fundies I had to do this because the `emake-tests` doesn't include this directory `ENIGMAsystem/SHELL/Universal_System` so what do you suggest? Moving to another test dir (outside emake-tests) or GLOBing everything inside `ENIGMAsystem/SHELL/Universal_System`?